package.json

```diff
@@ -31,8 +31,8 @@
     "bundle-umd": "microbundle --name h3 --format=umd",
     "bundle-cjs": "microbundle --format=cjs --no-compress",
     "bundle-es": "microbundle --format=es --no-compress",
-    "bundle-cjs-browser": "microbundle -o dist/browser --format=cjs --no-compress --alias ./out/libh3=./dist/libh3-browser",
-    "bundle-es-browser": "microbundle -o dist/browser --format=es --no-compress --alias ./out/libh3=./dist/libh3-browser",
+    "bundle-cjs-browser": "microbundle -o dist/browser --format=cjs --no-compress --alias ../out/libh3=$PWD/dist/libh3-browser",
+    "bundle-es-browser": "microbundle -o dist/browser --format=es --no-compress --alias ../out/libh3=$PWD/dist/libh3-browser",
```

Comment:
Will `--alias ../out/libh3=$PWD/dist/libh3-browser` behave unexpectedly when `$PWD` contains spaces?

Reply:
...Good question. Not sure what the best practice is here - though note that this only needs to run on developer machines and Travis, so mistakes here won't be an issue for end users.

Reply:
Updated to `$(printf '%q' $PWD)` to escape.

Reply:
Is that different from `\"../out/libh3=$PWD/dist/libh3-browser\"`?